templates/registration/upenn_start_2.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'base.html' %}
```

Comment:
@danaspiegel Just so that the form submits to upenn_start_2 instead of upenn_start